CHANGELOG.rst

```diff
@@ -11,11 +11,15 @@ Changelog
 * Added overrides to panels
 * Extend ``SeriesOverride`` options
 
+Changes
+-------
+
+* Fix Text panel (and add tests)
```

Comment:
Please could you add a warning that the fix has only been tested on > Grafana v8